scheduler/scheduler.go

```diff
@@ -575,36 +585,39 @@ func (s *scheduler) stopTask(id, source string) []serror.SnapError {
 	default:
 		// Group depndencies by the host they live on and
 		// unsubscirbe them since task is stopping.
```

Comment:
Would You mind to fix this typos -> "unsubscribe" and "dependencies" - I will really appreciate it.